server/ai_live_video.go

```diff
@@ -170,3 +172,54 @@ func startControlPublish(control *url.URL, params aiRequestParams) {
 		}
 	}()
 }
+
+func startEventsSubscribe(ctx context.Context, url *url.URL, params aiRequestParams) {
+	subscriber := trickle.NewTrickleSubscriber(url.String())
+
+	clog.Infof(ctx, "Starting event subscription for URL: %s", url.String())
+
+	go func() {
+		for {
+			clog.Infof(ctx, "Attempting to read from event subscription for URL: %s", url.String())
+			segment, err := subscriber.Read()
+			if err != nil {
+				clog.Infof(ctx, "Error reading events subscription: %s", err)
+				monitor.DeletePipelineStatus(params.liveParams.stream)
```

Comment:
I think we should hold on to the last known status of the stream. Only delete it when the stream is done, which is not "decided" here AFAICT. WDYT?

Reply:
As mentioned in the previous comment, commented out and added a todo